packages/preview/bamdone-aiaa/0.1.2/template/main.typ

```diff
@@ -0,0 +1,227 @@
+//***************************************************************
+// AIAA TYPST TEMPLATE
+// 
+// The author of this work hereby waives all claim of copyright
+// (economic and moral) in this work and immediately places it 
+// in the public domain; it may be used, distorted or 
+// in any manner whatsoever without further attribution or notice
+// to the creator. The author is not responsible for any liability 
+// from the usage or dissemination of this code.
+//
+// Author: Isaac Weintraub, Alexander Von Moll
+// Date: 03 DEC 2024
+// BAMDONE!
+//***************************************************************
+
+#import "@preview/bamdone-aiaa:0.1.2": *
+#import "@preview/droplet:0.3.1": dropcap
+
+#show: aiaa.with(
+    title: "Preparation of Papers for AIAA Technical Conferences",
+    bibliography-file: "refs.bib",
```

Comment:
The proper way to fix this is to take a `bibliography(...)` call directly as a parameter here and not just the file (see [this package](https://github.com/typst/packages/blob/main/packages/preview/charged-ieee/0.1.3/template/main.typ#L25) if you need a reference). If you just give a path here, it will be resolved relative to the package root and not to the user's project (in general, it is resolved relative to where `bibliography` is called), so the only possible path will be `refs.bib` and users won't be able to edit the contents of that file.

Reply:
I was able to fix this per your guidance. Thanks!